src/configure/mod.rs

```diff
@@ -96,11 +97,12 @@ impl Context {
 
         match success {
             Ok(_) => {
+                self.set_config(Config::NotifyAboutWrongPw, Some("1"))
+                    .await?;
                 progress!(self, 1000);
                 Ok(())
             }
             Err(err) => {
-                error!(self, "Configure Failed: {}", err);
                 progress!(self, 0);
```

Comment:
This extra statement was introduced in https://github.com/deltachat/deltachat-core-rust/pull/1539 and it led to "Configure Failed: could not connect" instead of "Maybe your pw is wrong, bla bla..." being shown in the dialog-box.